flow/connectors/utils/columns.go

```diff
@@ -4,51 +4,27 @@ import (
 	"slices"
 
 	"github.com/PeerDB-io/peer-flow/generated/protos"
-	"golang.org/x/exp/maps"
 )
 
 func TableSchemaColumns(schema *protos.TableSchema) int {
```

Comment:
This file should probably be removed entirely

Reply:
will do in a later PR, will require a lot of changes